### PR TITLE
Gears retracted when starting in-air.

### PR DESCRIPTION
**Pull request by NikolaiVChr**
:)

### CODE REVIEW

**Comment by NikolaiVChr** [comment]
Oh, I found a bug, do not merge yet.

**Comment by NikolaiVChr** [comment]
Okay, fixed the bug. But could not get it to start with retracted gears.
But now it will start with gears commanded up if in the air.

**Comment by abassign** [comment]
Ciao :) 

Forgive me for the delay, but unfortunately I had not seen that you had inserted the modification for the problem of the landing gear. Today I looked at your correction, but I cannot adopt it as I have already solved the problem in another way, I will explain how, also because the method I used is much more adherent to the philosophy with which I am making the G91. I'd like to understand if you can share it so that maybe we can collaborate on other things.

Meanwhile, being the command to place the landing gear in an open or closed position at the beginning of the execution (or re-execution after a positioning) a command that does not depend on the "mechanical" management of the landing gear, but on a state of the system. I prefer to insert the control in ... "G91-R1B_HD/G91-R1B-common.xml" to leave the landing gear code cleaner and do not carry out the control during the flight when the initial conditions are no longer necessary to evaluate.

Then, while searching for the solution, I noticed that the information of the initial state is present in: "/sim/presets/onground" which informs the system of the actual initial state, without having to evaluate the distance from the ground which is a solution good, but too tied to an external context that has not always been loaded at that precise moment. For example, if the terrain profile file is missing ... it becomes a problem ...

This is the solution I have adopted:

       <set-gear-down-script>
           <script>
                if(getprop("/sim/presets/onground")) {
                    setprop("controls/gear/gear-down", "true");
                    setprop("/controls/gear/brake-parking",1);
                } else {
                    setprop("controls/gear/gear-down", "false");
                }
            </script>
        </set-gear-down-script>

However, if you made any changes, please notify me about the github chat, so I can check with you immediately. I am very sorry that I was not careful, and for this I apologize again!

_Now I would like to ask you one thing:_

Have you ever run NASAL code in multiplayer?

The reason is to allow the plane to load the livery via canvas. Currently the G91 uses canvases to load liveries and is a very convenient, quick and flexible method. I would like to continue using it as the standad system of FGFS does not allow the same flexibility of use. Then there are several animations that would be much more precise if they could be run with NASAL.
Another problem is that if I had to use the liveries in a classic way, I would have to rearrange all the folders since I saw them the "Mirage-2000" code that starts like this:

File: Mirage-2000/Models/m2000-5.xml
```

<?xml version="1.0" encoding="iso-8859-1"?>
<!-- changelog 26/06/2017 Updated by Helijah -->
<PropertyList>
   <nasal>
    <load>
      <![CDATA[
      if(! contains(globals, "m2000_mp"))
      {
          var import = func (file, module) {
              file = string.normpath(file);
              var local_file = globals.io.dirname(caller()[2]) ~ file;
              var path = (globals.io.stat(local_file) != nil) ? local_file : resolvepath(file);
              globals.io.load_nasal(path, module);
          };
          import("Aircraft/Mirage-2000/Nasal/MP.nas", "m2000_mp");
      }
      var update = aircraft.overlay_update.new();
      update.add("Aircraft/Mirage-2000/Models/Liveries", "sim/model/livery/file");
      update.add("Aircraft/Mirage-2000/Models/Logos/", "sim/multiplay/generic/string");
```

As you can see, during the MP loading the code is executed ... I thought I would run the nasal file that loads the liveries with the canvas functions in the same way.
What do you think about it ?
Maybe if you want to try it too ... at least I start to share work with other friends and maybe to learn to be more attentive to those who give me a hand in the work.

**A note:**

The NASAL livery upload file is here:
../nasal/liveries.nas
It is executed at the start, but in MP it is not executed ... then obviously there is the problem of communicating the livery selected on the remote plane ... another problem ...
in short, there is still a lot of work to do;)

**Comment by NikolaiVChr** [comment]
I dont know why that file isnt loaded in MP.

I think you should ask 5H1N0B1, he has experience with loading files over MP. :)
